src/sentry/web/api.py

```diff
@@ -364,6 +365,10 @@ def get(self, request, **kwargs):
             response['X-Sentry-ID'] = response_or_event_id
         return response
 
+    def pre_normalize(self, data, helper):
```

Comment:
_nit and very opinionated_: `pre_process` might be a better name, since the other method is called `process` and it internally also calls `process_csp_report`.

Reply:
Not sure, it actually happens after `decode` and before `normalize`, but all of these three methods happen inside of `process` Hence the name.